.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I think this file can be deleted.  You can probably add `.vscode` to `.gitignore` too.